input/ig.xml

```diff
@@ -794,15 +794,20 @@
 				<generation value="markdown"/>
 			</page>
 			<page>
+				<nameUrl value="support.html"/>
+				<title value="Support"/>
+				<generation value="markdown"/>
+			</page>
+				<page>
 				<nameUrl value="downloads.html"/>
 				<title value="Downloads"/>
 				<generation value="markdown"/>
-			</page>
-			<page>
+				</page>
```

Comment:
This isn't a required change but be wary of changing the indenting on files - it shows up in the diff and adds to reviewer burden.

Reply:
Noted, will not amend in future.